src/lib/api.js

```diff
@@ -0,0 +1,42 @@
+import { buildSection } from "./character.js";
+
+export function createCharacter() {
+  const filterDropdown = document.querySelector(".header__filter");
+  const type = filterDropdown.value;
+  let url = getUrlFrom(type);
+
+  fetch(url)
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        console.log(response.status);
+      }
+    })
+    .then((dataObject) =>
+      dataObject.results.forEach((character) => {
+        buildSection(character);
+      })
+    );
+}
+
+function getUrlFrom(type) {
+  let url;
+  const search = document.querySelector(".header__search-form");
+  const searchInput = search.value;
+  if (type === "alive") {
+    url = `https://rickandmortyapi.com/api/character/?name=${searchInput}&status=alive`;
+  } else if (type === "dead") {
```

Comment:
Maybe you can save the base of the url as a variable and reduce some code repetition for the next 'if statements'...

Reply:
Thanks, great advice, I'll introduce a base url